specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -2589,6 +2589,51 @@
         "url": "https://docs.microsoft.com/rest/api/searchservice/Custom-analyzers-in-Azure-Search"
       }
     },
+    "NormalizerName": {
```

Comment:
Per past feedback from the Arch board, we should probably call this `LexicalNormalizerName`, since "normalizer" can be a pretty broad term in the context of Azure as a whole.

Reply:
Updated, thanks